app/metrics.yaml

```diff
@@ -4556,6 +4556,73 @@ tracking_protection:
       - android-probes@mozilla.com
     expires: 111
 
```

Comment:
just a drive by, don't think we want a new line here?

Reply:
That line was already there.
I didn't add it but I can remove it though there are other similar declarations.
Can cleanup all in a separate commit. ?

Reply:
Removed the newlines in a separate commit.